modules/openapi-generator/src/main/resources/Java/libraries/jersey2/JSON.mustache

```diff
@@ -27,8 +27,8 @@ public class JSON implements ContextResolver<ObjectMapper> {
   public JSON() {
     mapper = new ObjectMapper();
     mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    mapper.configure(DeserializationFeature.FAIL_ON_INVALID_SUBTYPE, false);
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true);
+    mapper.configure(DeserializationFeature.FAIL_ON_INVALID_SUBTYPE, true);
```

Comment:
@wing328 is there a way to toggle these back to false?

Reply:
I just wasted two hours on debugging after updating the OpenApiTools gradle plugin. This is a very breaking change and it was not mentioned by the changelog or release notes. This is also not in line with most of the other generator-libraries and there is no explanation for this change. Why was this changed silently?